# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libGLESv2/proc_table.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libGLESv2/proc_table.h
--- a/gfx/angle/checkout/src/libGLESv2/proc_table.h	Sat Sep 07 13:36:46 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,25 +0,0 @@
-//
-// Copyright 2017 The ANGLE Project Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-//
-// getProcAddress loader table:
-//   Mapping from a string entry point name to function address.
-//
-
-#ifndef LIBGLESV2_PROC_TABLE_H_
-#define LIBGLESV2_PROC_TABLE_H_
-
-#include <EGL/egl.h>
-#include <stddef.h>
-#include <utility>
-
-namespace egl
-{
-using ProcEntry = std::pair<const char *, __eglMustCastToProperFunctionPointerType>;
-
-extern ProcEntry g_procTable[];
-extern size_t g_numProcs;
-}  // namespace egl
-
-#endif  // LIBGLESV2_PROC_TABLE_H_